codalab/worker/docker_utils.py

```diff
@@ -220,7 +220,7 @@ def container_exists(container):
     try:
         client.containers.get(container.id)
         return True
-    except docker.errors.NotFound:
+    except docker.errors.ImageNotFound:
```

Comment:
This is for finding a `Container` not an `Image`. Can you double check which error trying to `get` a nonexistent container throws?